runtime/caml/sizeclasses.h

```diff
@@ -1,6 +1,6 @@
 /* This file is generated by tools/gen_sizeclasses.ml */
 #define POOL_WSIZE 4096
-#define POOL_HEADER_WSIZE 4
+#define POOL_HEADER_WSIZE 8
```

Comment:
This is a generated file, I think you need to change `tools/gen_sizeclasses.ml`

Reply:
I believe this might be an issue that's breaking the tests, because other values depend on `POOL_HEADER_WSIZE`